modules/mo_pytorch/mo_extensions/load/pytorch/hooks.py

```diff
@@ -112,14 +112,28 @@ def __init__(self, value):
             return forward_hook(Add(a), (self,), res)
 
     def __getitem__(self, n):
+        import numpy as np
+
+        value_ndim = self._value.ndim
+        begin_id = np.zeros(value_ndim, dtype=int)
+        end_id = np.zeros(value_ndim, dtype=int)
+        mask_arr = np.zeros(value_ndim, dtype=int)
+
+        for i in range(len(n)):
+            if isinstance(n[i], int):
```

Comment:
What other types we can have?

Reply:
tuple

Reply:
Do we know how to manage tuple in this way? There is no `else` here